changelogs/fragments/2091-add-loganalytics-callback-plugin.yml

```diff
@@ -0,0 +1,4 @@
+---
+add plugin.callback:
```

Comment:
Sorry, but this isn't needed. Explicit fragments are only needed for jinja2 test and filter plugins.

(CC @Akasurde)

Reply:
updated with "minor_changes"

Reply:
Please remove it completely. New plugins must not have changelog fragments (except for test and filter plugins, or when the PR modifies some shared code that was already there before).